package.json

```diff
@@ -65,6 +65,7 @@
     "fast-check": "^3.20.0",
     "glob": "^11.0.0",
     "madge": "^7.0.0",
+    "pkg-pr-new": "^0.0.17",
```

Comment:
Instead of installing this as a dependency, let'\s use `npx` with a version lock (`npx pkg-pr-new@0.0.17`) as this is purely a CI thing.